Ludum Dare 44/Assets/_Scripts/Enemy.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+
+public class Enemy : MonoBehaviour {
+    [SerializeField]
+    private EnemyObject enemy;
+
+    private int curHealth;
+
+    private bool isAlive = true;

```

Comment:
S: Get rid of or use getter (computer property). See line 24, where we dont update `isAlive`, now we're out of sync which is bad

Reply:
probably dont have to fix